### ISSUE TITLE
Give a better error message when an instructor tries to create a session that clashes with the recycle bin

### ISSUE THREAD

**Issue by damithc** [TEAMMATES/teammates#11300]
Context: an instructor tries to create a new session (e.g., by copying a session from another course) but it clashes with a session in the recycle bin.

The current error message:
![image](https://user-images.githubusercontent.com/1673303/126874844-9053293e-82a5-4f0c-a491-e3216d9cc225.png)

Suggestion: 
* reduce details in the error message
* a more end-user friendly message? e.g., `a session with the same name exists in the course ...`
* inform the user the clashing session is in the recycle bin (?) -- or at least mention the recycle bin in the error message.

**Comment by xyzhang00** [comment]
Hello, I would like to try working on this :)

**Comment by damithc** [comment]
Sure, go ahead @xyzhang00

**Comment by wkurniawan07** [comment]
This is not necessarily a session that "clashes with the recycle bin". This is a general issue for creating session with duplicate name within the course.

**Comment by damithc** [comment]
> This is not necessarily a session that "clashes with the recycle bin". This is a general issue for creating session with duplicate name within the course.

Yes, you are right. If a recycle bin clash can be distinguished easily, additional guidance can be given (because users may not be aware of that gotcha). But if not easy to distinguish, can simply include a caution about the recycle bin in all cases of this error message.

**Comment by xyzhang00** [comment]
<img width="724" alt="Screenshot 2021-07-25 at 5 48 48 PM" src="https://user-images.githubusercontent.com/54242896/126899841-87356285-478e-4bd0-bd9b-0c5f1469dc7c.png">
While fixing this issue, I realised that this is the error message when trying to create a new course under the same context. Would it be better to standardise these error messages?

**Comment by damithc** [comment]
> <img alt="Screenshot 2021-07-25 at 5 48 48 PM" width="724" src="https://user-images.githubusercontent.com/54242896/126899841-87356285-478e-4bd0-bd9b-0c5f1469dc7c.png">
> 
> While fixing this issue, I realised that this is the error message when trying to create a new course under the same context. Would it be better to standardise these error messages?

We can certainly mention the Recycle bin here as well. But probably the rest of the message will have to be different as the constraints are different for courses and sessions.
